content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -96,14 +96,7 @@ $ dvc exp show
 ```
 
 ```dvctable
- ───────────────────────────────────────────────────────────────────────────────────────────
-  neutral:**Experiment**              neutral:**Created**           metric:**loss**      metric:**acc**   param:**train.epochs**   param:**model.conv_units**
- ───────────────────────────────────────────────────────────────────────────────────────────
-  workspace               -              0.23657   0.9127   10             16
-  baseline-experiment     Sep 06, 2021   0.23657   0.9127   10             16
-  ├── 6d13f33 [cnn-64]    Sep 09, 2021   0.23385   0.9153   10             64
-  ├── 69503c6 [cnn-128]   Sep 09, 2021   0.23243    0.916   10             128
- ───────────────────────────────────────────────────────────────────────────────────────────
+$dvc-experiments-exp-show
```

Comment:
@iesahin I have only updated this one because the others are using the old `--include`/`--exclude` arguments (leftover from #3112 )

Reply:
For example, if we're to check in this line to Git, we'll need to run the script to populate the tables, right? My "creating a dependency" comment is about this.

Reply:
The actual table that replaces each of these placeholders (`${X}`) is also committed in Git (https://github.com/iterative/dvc.org/blob/083f0655b98bfb193a1fd96e00a4cb256328af9e/plugins/gatsby-remark-dvctable-filler/tables.js). 

There is no need to run the script, `yarn develop` will replace the placeholder with the corresponding table content.